### ISSUE TITLE
Potential accessibility issues with the Details (hidden text) component

### ISSUE THREAD

**Issue by JonathanCulling** [alphagov/govuk-design-system-backlog#273]
## What
> We have recently completed some accessibility research with Assistive Technology users and found a couple of potential issues with the Details (hidden text) component.

## Why
> 1. Visually impaired participants using screen readers (such as NVDA and Read&Write) did not understand how they were supposed to interact with the links, as they were not announced as links or bullets.
> 2. Two of our cognitively impaired participants with health conditions complained that the large yellow highlighted active area could potentially trigger pressure headaches.
![GDS standard potential issues 3](https://user-images.githubusercontent.com/124280348/216372358-70c54146-466b-46c8-8a1e-d1bdd958226a.png)

## Anything else
> I have included a slide from our report which covers both of these issues. Are there any tweaks we can make to this component to eliminate these issues for our audience?
